dev-docs/modules/weboramaRtdProvider.md

```diff
@@ -394,6 +395,37 @@ pbjs.que.push(function () {
 });
 ```
 
+An alternative version, using asset id instead url on contextual, can be found here:
```

Comment:
is this missing an "of"? i.e. "using asset id instead of url"

Reply:
You are right, this will be fixed soon

Reply:
done